### PR TITLE
Use a nameserver if specified in the environment variable CVDUPDATE_NAMESERVER

### PR DESCRIPTION
**Pull request by neodingo**

### CODE REVIEW

File: cvdupdate/cvdupdate.py [hunk 1]
```diff
@@ -369,7 +369,14 @@ def _query_dns_txt_entry(self) -> bool:
         self.logger.debug(f"Checking available versions via DNS TXT entry query of current.cvd.clamav.net")
 
         try:
-            answer = str(resolver.resolve("current.cvd.clamav.net","TXT").response.answer[0])
+            our_resolver = resolver.Resolver()
+            nameserver = os.environ.get("CVDUPDATE_NAMESERVER")
+
+            if nameserver != None:
+                our_resolver.nameservers = [nameserver]
+                self.logger.info("Using nameserver specified in CVDUPDATE_NAMESERVER: {nameserver}")
```
Comment (micahsnyder) [inline cvdupdate/cvdupdate.py:377]: This needs to be an f-string to print the nameserver:
```python
                self.logger.info(f"Using nameserver specified in CVDUPDATE_NAMESERVER: {nameserver}")
```

**Review by micahsnyder — CHANGES_REQUESTED** [review]
Neat feature, thanks for the PR. 

Can you please add a section in the README.md to describe how to use it?

File: cvdupdate/cvdupdate.py (hunk 1 above)
Comment (neodingo) [reply cvdupdate/cvdupdate.py]: f-string added, documentation updated.

**Review by micahsnyder — APPROVED** [review]
Looks good, seems to work nicely. Thanks!

**Pull request merged**